docs/pages/install/binaries.md

```diff
@@ -0,0 +1,98 @@
+
+## Supported Architectures
+
+Stone binaries are available for the following architectures:
+- **x86_64** (Linux and macOS)
+- **ARM64** (macOS and Linux)
+
+## Download and Install Binaries
+
+### For macOS (ARM64)
+
+1. Download the cpu_air_prover and cpu_air_verifier binaries:
+
+```bash
+wget https://github.com/dipdup-io/stone-packaging/releases/latest/download/cpu_air_prover-arm64 -O /usr/local/bin/cpu_air_prover
+wget https://github.com/dipdup-io/stone-packaging/releases/latest/download/cpu_air_verifier-arm64 -O /usr/local/bin/cpu_air_verifier
+```
+2. Make the binaries executable:
+
+```bash
+chmod +x /usr/local/bin/cpu_air_prover
+chmod +x /usr/local/bin/cpu_air_verifier
+```
+
+3. Move the binaries to /usr/bin for system-wide access (optional):
+
+```bash
+sudo mv /usr/local/bin/cpu_air_prover /usr/bin/
+sudo mv /usr/local/bin/cpu_air_verifier /usr/bin/
+```
+
+This step is optional but recommended if you want to make the binaries globally available without adding /usr/local/bin to your PATH.
+
+### For Linux (x86_64)
+
+1. Download the `cpu_air_prover` and `cpu_air_verifier` binaries:
+
+```bash
+sudo wget https://github.com/dipdup-io/stone-packaging/releases/latest/download/cpu_air_prover-x86_64 -O /usr/local/bin/cpu_air_prover
+sudo wget https://github.com/dipdup-io/stone-packaging/releases/latest/download/cpu_air_verifier-x86_64 -O /usr/local/bin/cpu_air_verifier
+```
+
+2. Make the binaries executable:
+
+```bash
+sudo chmod +x /usr/local/bin/cpu_air_prover
+sudo chmod +x /usr/local/bin/cpu_air_verifier
+```
+
+3. Move the binaries to /usr/bin for system-wide access (optional):
+
+```bash
+sudo mv /usr/local/bin/cpu_air_prover /usr/bin/
+sudo mv /usr/local/bin/cpu_air_verifier /usr/bin/
+```
+
+This step is actually optional but recommended if you want to make the binaries globally available without adding /usr/local/bin to your PATH.
+
+## Adding Stone Binaries to the System PATH
+
+If you prefer to leave the binaries in /usr/local/bin or any other folder, you can add that folder to your system’s PATH variable. This will ensure that the binaries can be executed from any directory.
+
+1. Open your shell configuration file (e.g., .bashrc for bash, .zshrc for zsh):
+
+```bash
+nano ~/.bashrc # for bash users
+nano ~/.zshrc # for zsh users
+```
+
+2. Add the following line to include /usr/local/bin in your PATH:
+
+```bash
+export PATH="/usr/local/bin:$PATH"
+```
+
+3. Save and close the file, then reload your shell to apply the changes:
+
+```bash
+source ~/.bashrc 
+# or
+source ~/.zshrc
+```
+
+This step ensures that binaries placed in /usr/local/bin can be accessed from anywhere in your system without needing to move them to /usr/bin.
+
+### Verifying Installation
+
+After placing the binaries or updating the PATH, you can verify the installation by running:
+
+```bash
+cpu_air_prover --help
```

Comment:
Does that work for you? What output do you get?

Reply:
it kinda worked but came up with some feedback which incudes a warning: SetUsageMessage() never called. let me try going around that and send in any update and progress

Reply:
just to mention that previously when i use the command `source ~/.bashrc` to reload my `.bashrc` it did errors such as "line 102: syntax error near unexpected token `then'
" and "line 114: syntax error: unexpected end of file
". so had to go into my shell configuration file and correct the syntax error and made sure indentations are fixed correctly before it reloaded properly